src/Api/General/GeneralAccount.php

```diff
@@ -2,17 +2,18 @@
 
 declare(strict_types=1);
 
-namespace Mailtrap\Api;
+namespace Mailtrap\Api\General;
 
+use Mailtrap\Api\AbstractApi;
 use Psr\Http\Message\ResponseInterface;
 
 /**
- * Class AbstractAccount
+ * Class GeneralAccount
```

Comment:
Let’s get rid of `General` prefix, since you already have it in the module path. Same goes to all classes in module
```suggestion
 * Class Account
```